dashboard/app/controllers/redirect_proxy_controller.rb

```diff
@@ -0,0 +1,26 @@
+require 'set'
+
+class RedirectProxyController < ApplicationController
+  include ProxyHelper
+
+  ALLOWED_HOSTNAME_SUFFIXES = %w(
+    bit.ly
+    ow.ly
+    t.co
+    tinyurl.com
+    Tr.im
```

Comment:
I think this should be lowercase to match the `.downcase` URL in `allowed_hostname?` https://github.com/code-dot-org/code-dot-org/blob/1e49ec6aa6bdae4755ab9397bb5acedcffa246a3/dashboard/app/helpers/proxy_helper.rb#L78